packages/lisk-p2p/src/peer_pool.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+/**
+ * The purpose of the PeerPool
+ */
+
+import { EventEmitter } from 'events';
+import http, { Server } from 'http';
+import querystring from 'querystring';
+import { Logger } from './p2p_types';
+import { Peer } from './peer';
+
+import socketClusterServer from 'socketcluster-server';
+
+export interface IPeerPoolConfig {
+	readonly blacklistedPeers?: ReadonlyArray<string>;
+	readonly logger: Logger;
+	readonly seedPeers: ReadonlyArray<string>;
+}
+
+export class PeerPool extends EventEmitter {
+	public httpServer: Server;
+	public logger: Logger;
+	public newPeers: Map<string, Peer>;
+	public scServer: any;
```

Comment:
`scServer` doesn't have the type from the library?

Reply:
No, the type definitions from the community were not usable.